app/design/frontend/Magento/elogic-theme/Magento_Cms/layout/cms_index_index.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="content">
+            <block class="Magento\Framework\View\Element\Template" name="main.slider"/>
+            <block class="Magento\Framework\View\Element\Template" name="categories"/>
+    </body>
+</page>
```

Comment:
Same comment as in for 2/14 task

Reply:
and another thing I've noticed: you need a close tag for `<referenceContainer>`:
```
<referenceContainer name="content">
            <block class="Magento\Framework\View\Element\Template" name="main.slider"/>
            <block class="Magento\Framework\View\Element\Template" name="categories"/>
</referenceContainer>
```